doc/api/child_process.markdown

```diff
@@ -194,20 +194,23 @@ replace the existing process and uses a shell to execute the command.*
   * `stderr` {Buffer}
 * Return: ChildProcess object
 
-The `child_process.execFile()` method is similar to [`child_process.exec()`][]
-except that it does not first spawn a shell. Rather, the specified `command` is
-spawned directly as a new process making it slightly more efficient than
-[`child_process.exec()`][]. The same options are support by both
-`child_process.exec()` and `child_process.execFile()`.
-
-    const exec = require('child_process').execFile;
-    const child = execFile('cat *.js bad_file | wc -l',
-      (error, stdout, stderr) => {
-        console.log(`stdout: ${stdout}`);
-        console.log(`stderr: ${stderr}`);
-        if (error !== null) {
-          console.log(`exec error: ${error}`);
-        }
+The `child_process.execFile()` function is similar to [`child_process.exec()`][]
+except that it does not spawn a shell. Rather, the specified executable `file`
+is spawned directly as a new process making it slightly more efficient than
+[`child_process.exec()`][].
+
+The same options as `child_process.exec()` are supported. If a `file` path
+contains no slash delimiters, `child_process.execFile()` searches for a matching
```

Comment:
Instead of "slash delimiters", I would say "path separators", "path delimiters", or just "slashes".

From a higher level, this might not be the best place to explain how `PATH` lookup works, as it's not specific to `execFile()`.